pkg/init/init

```diff
@@ -2,32 +2,11 @@
 
 setup_console() {
 	tty=${1%,*}
-	speed=${1#*,}
 	inittab="$2"
-	securetty="$3"
-	line=
-	term="linux"
-	[ "$speed" = "$1" ] && speed=115200
-
-	case "$tty" in
-	ttyS*|ttyAMA*|ttyUSB*|ttyMFD*)
-		line="-L"
-		term="vt100"
-		;;
-	tty?)
-		line=""
-		speed="38400"
-		term=""
-		;;
-	esac
 	# skip consoles already in inittab
 	grep -q "^$tty:" "$inittab" && return
 
 	echo "$tty::once:cat /etc/issue" >> "$inittab"
```

Comment:
yeah i think thats ok. We can always add a `whale` package for the desperate 🐳. The assumption is if you don't have its probably not something you are going to be looking at much.